app/dashboard/views.py

```diff
@@ -2167,6 +2177,38 @@ def new_bounty(request):
     return TemplateResponse(request, 'bounty/new.html', params)
 
 
+@csrf_exempt
+def get_suggested_contributors(request):
+    previously_worked_developers = []
+    keywords = request.GET.get('keywords', '').split(',')
+    if request.user.is_authenticated:
+        previously_worked_developers = BountyFulfillment.objects.prefetch_related('bounty')\
+            .filter(
+                bounty__bounty_owner_github_username__iexact=request.user.profile.handle,
+                bounty__idx_status='done'
+            ).values('fulfiller_github_username').annotate(fulfillment_count=Count('bounty')) \
+            .order_by('-fulfillment_count')
+        
+    keywords_filter = Q()
+    for keyword in keywords:
```

Comment:
I had changed this query to work with more than one keyword and also changed the front end to send is as param, since is a GET and there is no body data.